django/core/validators.py

```diff
@@ -252,7 +309,8 @@ def __eq__(self, other):
 
 validate_email = EmailValidator()
 
-slug_re = _lazy_re_compile(r"^[-a-zA-Z0-9_]+\Z")
+
+slug_re = _lazy_re_compile(r'^[-a-zA-Z0-9_]+\Z')
```

Comment:
this might be something they won't want in this PR since it's unrelated to the rest